dev-python/test/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>python@gentoo.org</email>
+	</maintainer>
```

Comment:
Do I dare to ask?

By chance is this an ALLARCHES since only python modules are installed?

Reply:
Technically, yes. However, we are going to lock-step stabilize it with `dev-lang/python` anyway.